examples/common/chip-app-server/RendezvousServer.cpp

```diff
@@ -27,6 +32,8 @@ using namespace ::chip::DeviceLayer;
 
 namespace chip {
 
+extern SecureSessionMgrBase & SessionManager();
```

Comment:
I'd prefer you included a header instead of re-declaring.  can we do that?

Reply:
@rwalker-apple Agree. For some reason SessionManager() wasn't put in the header and its users re-declare it, but I can change that.